themes/vocabulary_theme/templates/page-with-toc.html

```diff
@@ -4,9 +4,9 @@
 
 {% block body %}
   <div class="code-guidelines">
-    <header class="header">
+    <header class="header" aria-labelledby="code-guidelines-title">
```

Comment:
unsure this relationship makes sense. the headings are already semantic landmarks. I don't see why binding the header to the h1 is useful here?